packages/wrangler/src/pages/functions/buildWorker.ts

```diff
@@ -367,6 +372,39 @@ export function buildNotifierPlugin(onEnd: () => void): Plugin {
 	};
 }
 
+/**
+ *
+ */
+export function additionalModulesWriterPlugin(
+	modules: CfModule[],
+	entry: Entry,
+	onEnd: () => void
+): Plugin {
+	return {
+		name: "wrangler-additional-modules-writer",
```

Comment:
Nit: could this be `wrangler-dependent-modules-writer`? The word `additional` might be misleading.

Reply:
oh, much better than the name I went for! I like it! I like it a lot!